lib/rspec/core/configuration.rb

```diff
@@ -1412,13 +1415,32 @@ def self.delegate_to_ordering_manager(*methods)
       #     end
       #   end
       #
-      #   describe MyClass, :order => :reverse do
+      #   RSpec.describe 'MyClass', :order => :reverse do
       #     # ...
       #   end
       #
       # @note Pass the symbol `:global` to set the ordering strategy that
       #   will be used to order the top-level example groups and any example
       #   groups that do not have declared `:order` metadata.
+      #
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.register_ordering :global do |examples|
+      #       acceptance, other = examples.partition do |example|
+      #         example.metadata[:type] == :acceptance
```

Comment:
I agree, though this is a special case.

First, Rails users likely won't use ordering anyway, except for random/seed. A "trivial" ordering example isn't practical. A practical example won't be trivial. Custom ordering in RSpec simply is a bit tricky right now.

To prove the point about complex ordering being non-trivial, consider this "simple example":

```ruby
examples.partition { |example|  example.metadata[:type] != :acceptance }.flatten(1)
```

It's one line, simple and readable yet ... not a good example.

Sometimes a longer and more elaborate example is much simpler to understand and use. I think this is exactly the case here.